src/validation.cpp

```diff
@@ -4303,7 +4304,15 @@ bool LoadMempool(void)
                 if (state.IsValid()) {
                     ++count;
                 } else {
-                    ++failed;
+                    // mempool may contain the transaction already, e.g. from
+                    // wallet(s) having loaded it while we were processing
+                    // mempool transactions; consider these as valid, instead of
+                    // failed, but mark them as 'already existed'
+                    if (mempool.exists(tx->GetHash())) {
```

Comment:
But REJECT_DUPLICATE is set if it's in the mempool which happens to be what's tested here. So it should be?
```cpp
} else if (state.GetRejectCode() == REJECT_DUPLICATE && state.GetRejectReason() == "txn-already-in-mempool") {
```